lib/scarpe/wv/edit_line.rb

```diff
@@ -2,7 +2,7 @@
 
 module Scarpe::Webview
   class EditLine < Drawable
-    attr_reader :text, :width
+    attr_reader :text, :width, :stoke, :font, :tooltip
```

Comment:
You probably want :stroke rather than :stoke here.

Reply:
I am so sorry for making such a mistake, it's embarrassing that even after checking the changed files many times I didn't notice it lol, but now that PR is merged I can't change it, I am thinking of making this change in next pull request I raise because I think making a pull request for such a small mistake isn't a good idea, but if it causes any trouble I can make a pull request now.

Reply:
Sorry, I should have thought of that and given you a chance to fix it. Not a big deal -- we use these attributes on Webview fairly rarely. I'll fix it now.